src/centriamTable/table.scss

```diff
@@ -42,6 +42,18 @@ $stripe-color: #FBFBFB;
         margin: 0 .7em;
       }
 
+      &.row-count {
+        left: 220px;
+        position: absolute;
+        font-family: sans, 'FontAwesome';
+        span {
+          span {
```

Comment:
second span probably isn't needed. we have div.row-count span. Not sure what this second span is targeting. 
Also 1em does not change the font size at all. 

Addtionally, realize this does not get used outside of this application at all. This is overwritten by whoever consumes this page.

Reply:
so the reason for the internal span is because when I pass a <NumberFormat> through in the prop, it passes it wrapped in <span> (and I dont think I can prevent this from happening, or I don't know how to strip it off). It appears to me that the font-size 1.4em gets applied twice to the count because of this, and results in a number that is larger than the other text. By setting that 1.0em it stops it from applying the 1.4em twice and then it looks the same as when I pass {1234} (an int) through as the count property